src/main/java/com/khutircraftubackend/product/ProductService.java

```diff
@@ -85,6 +89,10 @@ public ProductEntity patchProduct(Long productId, ProductUpdateRequest request,
         if (request.description() != null) {
             existingProduct.setDescription(request.description());
         }
+        if(request.categoryId() != null) {
+            existingProduct.setCategory(categoryRepository.findById(request.categoryId()).orElseThrow(() ->
+            new CategoryNotFoundException(CategoryExceptionMessages.CATEGORY_NOT_FOUND)));
+        }
```

Comment:
any validations for product name or something?

Reply:
For now - yes.

Reply:
hi,
do you mean, for now we do not validate product's name, right?

Reply:
I don't understand.